Tests/SwiftParserTest/translated/MatchingPatternsTests.swift

```diff
@@ -612,125 +612,122 @@ final class MatchingPatternsTests: ParserTestCase {
-      case _borrowing x.member: // parses as var introducer surrounding postfix expression (which never is valid)
+      case borrowing x.member: // parses as var introducer surrounding postfix expression (which never is valid)
         break
       }
       """,
       substructure: PatternSyntax(
         ValueBindingPatternSyntax(
-          bindingSpecifier: .keyword(._borrowing),
+          bindingSpecifier: .keyword(.borrowing),
           pattern: ExpressionPatternSyntax(
             expression: MemberAccessExprSyntax(
               base: DeclReferenceExprSyntax(baseName: .identifier("x")),
               declName: DeclReferenceExprSyntax(baseName: .identifier("member"))
             )
           )
         )
-      ),
-      experimentalFeatures: .borrowingSwitch
+      )
     )
     assertParse(
       """
       switch 42 {
-      case let _borrowing: // parses as let binding named '_borrowing'
+      case let borrowing: // parses as let binding named 'borrowing'
         break
       }
       """,
       substructure: PatternSyntax(
         ValueBindingPatternSyntax(
           bindingSpecifier: .keyword(.let),
-          pattern: PatternSyntax(IdentifierPatternSyntax(identifier: .identifier("_borrowing")))
+          pattern: PatternSyntax(IdentifierPatternSyntax(identifier: .identifier("borrowing")))
         )
       ),
       experimentalFeatures: .borrowingSwitch
     )
     assertParse(
       """
       switch 42 {
-      case _borrowing + _borrowing: // parses as expr pattern
+      case borrowing + borrowing: // parses as expr pattern
         break
       }
       """,
-      substructure: ExprSyntax(DeclReferenceExprSyntax(baseName: .identifier("_borrowing"))),
+      substructure: ExprSyntax(DeclReferenceExprSyntax(baseName: .identifier("borrowing"))),
       experimentalFeatures: .borrowingSwitch
     )
     assertParse(
       """
       switch 42 {
-      case _borrowing(let _borrowing): // parses as let binding named '_borrowing' inside a case pattern named 'borrowing'
+      case borrowing(let borrowing): // parses as let binding named 'borrowing' inside a case pattern named 'borrowing'
         break
       }
       """,
       substructure: PatternSyntax(
         ValueBindingPatternSyntax(
           bindingSpecifier: .keyword(.let),
-          pattern: PatternSyntax(IdentifierPatternSyntax(identifier: .identifier("_borrowing")))
+          pattern: PatternSyntax(IdentifierPatternSyntax(identifier: .identifier("borrowing")))
         )
       ),
       experimentalFeatures: .borrowingSwitch
     )
     assertParse(
       """
       switch 42 {
-      case {}(_borrowing + _borrowing): // parses as expr pattern
+      case {}(borrowing + borrowing): // parses as expr pattern
         break
       }
       """,
-      substructure: ExprSyntax(DeclReferenceExprSyntax(baseName: .identifier("_borrowing"))),
+      substructure: ExprSyntax(DeclReferenceExprSyntax(baseName: .identifier("borrowing"))),
```

Comment:
Would it make sense to continue having tests for `_borrowing`?

Reply:
My hope is that the warning I added in apple/swift#72996 will get the early adopters over the new syntax so I can clean up the temporary syntax in a follow up commit.